examples/with-react-hook-form/README.md

```diff
@@ -0,0 +1,18 @@
+# Minimal Next.js App Router example for UploadThing
```

Comment:
Title should probably mention react-hook-form

Reply:
i crawled all the repo looking for a react-hook-form implementation, thank you for the suggestion